observer/query.go

```diff
@@ -0,0 +1,111 @@
+package observer
```

Comment:
This allows us to run the observer under F3 passive testing infra as a HTTP server and CURL it with SQL statements, e.g.:

```
$ curl -X POST -d 'select distinct sender from messages'  https://where-it-is-gonna-run/query
```